code/API_definitions/geofencing-subscriptions.yaml

```
@@ -923,15 +925,15 @@ components:
       type: string
       description: |
         - NETWORK_TERMINATED - API server stopped sending notification
-        - SUBSCRIPTION_CREATION_ABORTED - Subscription cannot be created because the specified area cannot be managed (for asynchronous subscription creation)
+        - SUBSCRIPTION_UNPROCESSABLE - Subscription cannot be processed because the specified area cannot be managed (for asynchronous subscription creation)
```

```suggestion
        - SUBSCRIPTION_UNPROCESSABLE - Subscription cannot be processed due to some reason, e.g. because the specified area cannot be managed. Useful for asynchronous subscription creation.
```

It can be generalized to any reason. Termination description can provide the details.